test-harness/specs/E2E-Accept-Content-Negotiation-AC-6.oas2.txt

```diff
@@ -24,4 +24,4 @@ content-type: application/problem+json
 content-length: 0
 Connection: keep-alive
 
-{"type":"https://stoplight.io/prism/errors#NOT_ACCEPTABLE","title":"The server cannot produce a representation for your accept header","status":406,"detail":"Unable to find content for application/json"}
+{"type":"https://stoplight.io/prism/errors#NOT_ACCEPTABLE","title":"The server cannot produce a representation for your accept header.","status":406,"detail":"Unable to find content for application/json"}
```

Comment:
I'm not sure if a title should have a full stop?

Reply:
Yeah I think you're right. I'll delete it everywhere.